src/Features/Core/Portable/ImplementInterface/ImplementHelpers.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using Microsoft.CodeAnalysis.LanguageService;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.ImplementInterface
+{
+    internal static class ImplementHelpers
+    {
+        public static ImmutableArray<ISymbol> GetDelegatableMembers(
+            Document document,
+            INamedTypeSymbol namedType,
+            Func<ITypeSymbol, bool> includeMemberType,
+            CancellationToken cancellationToken)
+        {
+            var syntaxFacts = document.GetRequiredLanguageService<ISyntaxFactsService>();
+
+            var fields = namedType.GetMembers()
+                .OfType<IFieldSymbol>()
+                .Where(f => !f.IsImplicitlyDeclared)
+                .Where(f => includeMemberType(f.Type))
+                .ToImmutableArray();
+
+            var properties = namedType.GetMembers()
+                .OfType<IPropertySymbol>()
+                .Where(p => !p.IsImplicitlyDeclared && p.Parameters.Length == 0 && p.GetMethod != null)
+                .Where(p => includeMemberType(p.Type))
+                .ToImmutableArray();
```

Comment:
up to here is hte same code as before.  below is the code that adds support for PC parameters.